### ISSUE TITLE
Not sure if emulator is set up correctly

### ISSUE THREAD

**Issue by SundaShep** [visualboyadvance-m/visualboyadvance-m#795]
I'm not a programmer myself but I recently installed VBA-m from this site so I could play a rom hack but I'm really not sure if it set up correctly. I've been using older versions and they always had a bunch of files I could unzip but this download is just the exe, but it seems to run  perfectly fine. I know it's probably a stupid question but I'm not sure if I'm missing files or if its just my inexperience in this type of thing. I'll attach a picture of the folder i unzipped the file too.
![Screenshot 2021-01-03 204947](https://user-images.githubusercontent.com/76932125/103503139-3faded00-4e08-11eb-985b-d4bff345aaf4.png)

**Comment by rkitover** [comment]
Yeah there is just the exe, that's all you need. Please use the nightly from here:

http://win.vba-m.com/nightly
